### DISCUSSION TITLE
# of Renard channels?

### DISCUSSION THREAD

**Discussion by Kirkblum** [forkineye/ESPixelStick#465] [Q&A]
Anyone have any ideas how many channels of Renard serial output is supported by the code for the 8266 and the 32??  Is a Renard channel similar to a pixel channel so basically we get 680 Renard channels out of a single 8266 or are we more serial channel limited for Renard protocol and therefore the # is higher?  Is there an e1.31 SACN limitation.  What about the ESP32 - is it faster so can handle more channels or are we again SACN or serial baud rate limited??

THX.

**Comment by MartinMueller2003** [comment]
The renard channels run at a much slower speed. Yes we can support up to 3000 channels of renard data. However updating that many channels over a single renard port would to multiple seconds. The practical limit for renard ports is in the 200 channel range when you run the ports at 200Kbps or higher. If you run at the default 57600bps rate you can get about 100 channels with a 50ms refresh rate.

While renard channels do not have fancy pixel features associated with them (zig zag/groups etc...) there is a Renard controller that converts Renard channels to WS281x data (PX1).
